app/styles/app.scss

```diff
@@ -438,4 +438,34 @@ li[aria-expanded=true] ul[role=menu].dropdown {
   border-radius: 0;
 }
 
-/* end hotfix */
\ No newline at end of file
+/* end hotfix */
+
+.octane-feature-summary-table {
```

Comment:
any table css should go to github.com/ember-learn/ember-styleguide please.

Reply:
what does that look like for smaller screens? does the ember-styleguide have any un-implemented tables? last I looked through the docs, there was nothing about tables?